applications/climatology/climatology.ipynb

```diff
@@ -0,0 +1,3533 @@
+{
```

Comment:
<p>Just a suggestion -- I'd consider adding <a href="https://docs.xarray.dev/en/stable/user-guide/time-series.html" target="_blank">https://docs.xarray.dev/en/stable/user-guide/time-series.html</a> here too, since it's cited elsewhere in the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>